Gemfile

```diff
@@ -53,3 +56,9 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'devise'
+gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'master'
+gem 'rubocop', require: false
```

Comment:
You are missing the rubocop files. Create `.pernix-rubocop.yml` and `.rubocop.yml` files I'll share with you the content of each file.

Reply:
Done